src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -117,6 +120,14 @@ message InterfaceConfigProto {
   repeated RetryParamsDefinitionProto retry_params_def = 31;
 }
 
+message SmokeTestConfigProto {
```

Comment:
Are we also going to include a way to check the return values? Will that be in a followup PR?